doc/data-structures.rst

```diff
@@ -401,6 +401,20 @@ operations keep around coordinates:
     list(ds[['x']])
     list(ds.drop('temperature'))
 
+When working with a dataset containing many variables, it can be useful to
+select a subset of variables. Passing a list of variables to the
+:py:meth:`~xarray.Dataset.get` method returns a ``Dataset`` with only those
+variables:
+
+.. ipython:: python
+
+    ds['temperature_double'] = (('x', 'y', 'time'), temp * 2)
+    ds['precipitation_double'] = (('x', 'y', 'time'), precip * 2)
+
+    list(ds.get(['temperature', 'temperature_double']))
```

Comment:
Why `list`?

Reply:
See above. I wasnt sure what to make of this either but kept if for consistency.